### DISCUSSION TITLE
SAE Manuelle Installation => Failed to start smartapplianceenabler.service: Unit mosquitto.service not found.

### DISCUSSION THREAD

**Discussion by jhartlep** [camueller/SmartApplianceEnabler#579] [Installation und Betrieb]
Moin,
ich habe mir gerade auf meinem Proxmox einen Ubuntu LXC hochgezogen und dort die Schritte der manuellen Installation abgearbeitet - allerdings ohne MQTT, da ich in meinem Netzwerk schon einen MQTT-Broker betreibe.

In der Datei Appliances.xml habe ich meinen MqttBroker mit host/port/username/password eingetragen.

Versuche ich nun, SAE zu starten (mit `sudo systemctl start smartapplianceenabler`), erhalte ich folgenden Fehler in der Konsole:

`Failed to start smartapplianceenabler.service: Unit mosquitto.service not found.`

Klar, in dem LXC gibt es einen mosquitto.service. Dafür habe ich ja die Appliances.xml. Aber warum startet das Ding nicht?
Leider wird unter `/var/log/` auch noch kein Logeintrag geschrieben...

Hat hier jemand eine Idee?

P.S.
LXC-Image: Ubuntu 22.04
SAE-Version: 2.3.1

P.P.S.
Ich habe in der Update-Dokumentation den Punkt gefunden, dass hier der Broker zwingend da sein muss... Wenn ich lokal einen zusätzlichen Broker installiere (Aber in der Config meinen eigenen Broker drin lasse), dann startet SAE. Muss man das hier tatsächlich so machen mit dem lokalen Broker?

**Comment by camueller** [comment]
Lösch doch in der Datei `/etc/systemd/system/smartapplianceenabler.service` die Zeile:
```
Requires=mosquitto.service
```
Dann sollte der SAE problemlos starten und Du kannst in den Einstellungen konfigurieren, wie Dein vorhandener MQTT-Broker erreichbar ist.

**Reply by jhartlep** [reply]
Zu einfach ^^
Hat geklappt. Vielen Dank für den Tipp...
